CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.2.0
+- Add support for diagnostic logs from Azure Functions
+- Update application property field to remove the gcm/pub/pvm prefix
+
```

Comment:
- lower case the resource group name value to align with standard casing for log properties